meson.build

```diff
@@ -1205,141 +1157,88 @@ if have
         conf.set_quoted('SMACK_DEFAULT_PROCESS_LABEL', get_option('smack-default-process-label'))
 endif
 
-want_polkit = get_option('polkit')
-install_polkit = false
-install_polkit_pkla = false
-if want_polkit != 'false' and not skip_deps
-        install_polkit = true
-
-        libpolkit = dependency('polkit-gobject-1',
-                               required : false)
-        if libpolkit.found() and libpolkit.version().version_compare('< 0.106')
-                message('Old polkit detected, will install pkla files')
-                install_polkit_pkla = true
-        endif
+libpolkit = dependency('polkit-gobject-1',
+                       required : get_option('polkit'))
```

Comment:
The dependency object is not passed to executables or libraries, so, we can downgrade the required level like we did previously. Maybe, the below should work.
`required : get_option('polkit').allowed()`

Reply:
I see now that the dependency is just used to check for old polkit. Behavior should now match the old one.